Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM amazonlinux:2017.03
```

Comment:
wondering if we should do a copy and call it mozilla-gold or whatever. maybe later, though.

Reply:
Later... yes.  Right now this mirrors the dev environment for the apex projects.